templates/partials/page-footer.hbs

```diff
@@ -41,4 +41,11 @@
       </a>
     </li>
   </ul>
+  <ul class="grav-c-list-horizontal">
+    <li>
+      <a href="/privacy">
```

Comment:
Please add the trailing slash (`/privacy/`), to avoid an unnecessary 301 redirect from `/privacy` to `/privacy/`.

Reply:
Thanks 👍